spark3/src/test/java/org/apache/iceberg/spark/source/TestSparkCatalog.java

```diff
@@ -31,7 +30,14 @@
 
   @Override
   public Table loadTable(Identifier ident) throws NoSuchTableException {
-    TestTables.TestTable table = TestTables.load(Spark3Util.identifierToTableIdentifier(ident).toString());
-    return new SparkTable(table, false);
+    String[] parts = ident.name().split("\\$", 2);
```

Comment:
I probably need to add a comment here. This is also for testing purposes until we consume Spark 3.2.